gatsby/content/blog/2019/10/2019-10-03-synapse-1.4.0-released.mdx

```diff
@@ -0,0 +1,248 @@
+ * If you have configured custom templates, then eight new templates must be added to your templates directory.
+
+If either apply to you failure to act will break your installation. Full details can be found in the [upgrade notes](https://github.com/matrix-org/synapse/blob/master/UPGRADE.rst#upgrading-to-v140).
+
+## So what has changed?
+
+It is possible for a user to associate an email address or phone number with their account, for a number of reasons:
+
+ * For use when logging in, as an alternative to the user id.
+ * In the case of email, as an alternative contact to help with account recovery.
+ * In the case of email, to receive notifications of missed messages.
+
+Before an email address or phone number can be added to a user's account, or before such an address is used to carry out a password-reset, Synapse must confirm the operation with the owner of the email address or phone number. It does this by sending an email or text giving the user a link or token to confirm receipt. This process is known as '3pid verification'. ('3pid', or 'threepid', stands for third-party identifier, and we use it to refer to external identifiers such as email addresses and phone numbers.)
+
+Previous versions of Synapse delegated the task of 3pid verification to an identity server by default. In most cases this server is vector.im or matrix.org.
+
+In Synapse 1.4.0, for security and privacy reasons, the homeserver will no longer delegate this task to an identity server by default. Instead, the server administrator will need to explicitly decide how they would like the verification messages to be sent.
+
+In the medium term, the vector.im and matrix.org identity servers will disable support for delegated 3pid verification entirely. However, in order to ease the transition, they will retain the capability for a limited period. Delegated email verification will be disabled on Monday 2nd December 2019 (giving roughly 2 months notice). Disabling delegated SMS verification will follow some time after that once SMS verification support lands in Synapse.
+
+Once delegated 3pid verification support has been disabled in the vector.im and matrix.org identity servers, all Synapse versions that depend on those instances will be unable to verify email and phone numbers through them. There are no imminent plans to remove delegated 3pid verification from Sydent generally. (Sydent is the identity server project that backs the vector.im and matrix.org instances).
+
+## Why is this necessary?
+Prior to 1.4.0, the identity server was providing two related-but-separate functions:
+
+a directory for users to publish their contact details and to look up their contacts by their email addresses and phone numbers.
+a "trusted third party communications network delegate", providing SMS- and email-sending functionality to all homeservers for registration and password reset.
```

Comment:
![Screen Shot 2019-10-03 at 12 07 04 PM](https://user-images.githubusercontent.com/5855073/66148357-51d22780-e5d6-11e9-87b1-eab2a530562c.png)

This doesn't look great in the rendered version. You probably need some bullet points or a numbered list.